src/v/cluster/controller.json

```diff
@@ -165,6 +166,11 @@
             "name": "set_partition_shard",
             "input_type": "set_partition_shard_request",
             "output_type": "set_partition_shard_reply"
+        },
+        {
+            "name": "alter_client_quotas",
+            "input_type": "client_quota::alter_quotas_request",
+            "output_type": "client_quota::alter_quotas_response"
```

Comment:
do we need a feature flag for this?

Reply:
Other examples of similar code (eg. [plugin_frontend.cc#L118-L139](https://github.com/redpanda-data/redpanda/blob/6094a4eaefd7b057747e1d463a4bcf3bb88207c2/src/v/cluster/plugin_frontend.cc#L118-L139)) don't seem to have this, so I am tempted to say we don't need one here either. We can just accept that in a mixed-version cluster, these requests will time out until the cluster is fully upgraded.

Reply:
sgtm